### ISSUE TITLE
[Feature Request] Is there a way to load test Postman Newman Scripts / API tests ? 

### ISSUE THREAD

**Issue by rashmipatil** [microsoft/azure-load-testing#10]
[Feature Request] Is there a way to load test Postman Newman Scripts / API tests ? In our organisation we use Postman to test API functionality, it would be great if we can reuse those scripts to load test our api's rather than having to convert them to JMeter to make it work in Azure Load Testing Feature ..

**Comment by kaumishra** [comment]
Hi @rashmipatil If I understood it correctly you would like to import an API collection and directly run on our service? This is a high priority item on our roadmap. Let me check on the timeline and get back.

**Comment by rashmipatil** [comment]
Yes, I am looking at importing an API collection and running a load test on
the collection using azure load testing service. Glad to know it's a
priority, will you be able to inform me about the timeline of this
functionality? When are you planning on delivering it?

Thank you

**Comment by adiazcan** [comment]
Hello! Do you have a plan for this feature on the roadmap? We also use Postman collection for testing our API and we want to use Azure Load Testing to do out load testing.

**Comment by ninallam** [comment]
@adiazcan Thanks for reaching out. Unfortunately we didn't see many customers asking for this, so we couldn't prioritize it. We'll watch out for more feedback so that we can plan properly.
